### PR TITLE
notifications fixes

### PR DESCRIPTION
**Pull request by arkadius**
- (1.9 regression) states wasn't refreshed after cancel finish/fail
- (was in 1.8) "state" state wasn't refreshed after finish (was only after fail)
- (was in 1.8) Notification "Deployment of ... failed ..." was presented even for "cancel" action

### CODE REVIEW

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/Notification.scala [hunk 1]
```diff
@@ -13,17 +15,23 @@ import pl.touk.nussknacker.ui.notifications.DataToRefresh.DataToRefresh
                                    toRefresh: List[DataToRefresh])
 
 object Notification {
-  def deploymentFailedNotification(id: String, name: ProcessName, failureMessageOpt: Option[String]): Notification = {
-    Notification(id, Some(name), s"Deployment of ${name.value} failed" + failureMessageOpt.map(" with reason: " + _).getOrElse(""),
+  def deploymentActionFailedNotification(id: String, actionType: ProcessActionType, name: ProcessName, failureMessageOpt: Option[String]): Notification = {
```
Comment (lciolecki) [inline designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/Notification.scala:18]: In fact, these aren't only deployment actions. Hence, actionFaildNotification?

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala [hunk 2]
```diff
@@ -146,10 +146,10 @@ extends Repository[F] with EspTables with CommentActions with ProcessActionRepos
     run(query.result.map(_.toSet))
   }
 
-  def getUserActionsAfter(user: LoggedUser, possibleStates: Set[ProcessActionState], actionType: ProcessActionType, limit: Instant): F[Seq[(ProcessActionEntityData, ProcessName)]] = {
+  def getUserActionsAfter(user: LoggedUser, possibleStates: Set[ProcessActionState], possibleActionTypes: Set[ProcessActionType], limit: Instant): F[Seq[(ProcessActionEntityData, ProcessName)]] = {
```
Comment (lciolecki) [inline designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala:149]: I know it's after the previous change, but I think the order of params here should be different: possibleActionTypes -> possibleStates.

**Review by lciolecki — APPROVED** [review]
Nice :)

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/Notification.scala (hunk 1 above)
Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/Notification.scala:18]: Good point, on this level we can generalize it

Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/Notification.scala]: done

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala (hunk 2 above)
Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala]: done

**Review by gadomsky — APPROVED** [review]

**Pull request merged**